### ISSUE TITLE
No keyboard control

### ISSUE THREAD

**Issue by neilmispelaar** [NeoCoast/vue-cascader-select#2]
## Expected Behavior

For accessibility purposes you should be able to use the tab key to focus control to the component and then use the keyboard arrow controls and the enter key to select an object (or esc or tab to move away from the object). 
 
## Actual Behavior

Right now the only way to control the component is with a mouse, which would fail WCAG 2.0

## Steps to Reproduce the Problem

  1. Open the page
  2. Use the tab key to tab through the page
  3. The tab focus will not stop at the component but rather skips over the component and focuses on the next focusable item on the page

## Specifications

 - Platform: Microsoft Edge on Mac OS

**Comment by Tintef** [comment]
@neilmispelaar Thanks for bringing this issue to the table.

I'm looking around on the best approach for this, what do you think about:
- Press tab to access the input
- Press enter to open the first menu
- Now that the menu is opened, you can navigate with the arrows (up and down to move through options and left right to move through menus) and with enter you can select an option
- Escape to close the cascader select

I found that only using tab is not actually possible, as you can select an option of a submenu.

**Comment by neilmispelaar** [comment]
I think that makes perfect sense. One thing I would add: 

After tabbing to access the input: 

- Use Spacebar or Enter to open the first menu 
- Use arrow keys to navigate (as you mentioned - up down left right)
- Use Spacebar or Enter to make a selection
- Escape or click outside to close the menu

Backgrounder: 

Spacebar (in addition to enter) is a standard keyboard control that users can expect to use when making selections. The following link has some expected user behaviours under the "Keyboard Testing" section.
https://webaim.org/techniques/keyboard/

**Comment by Tintef** [comment]
@neilmispelaar Hey, just wanted to let you know that the changes on #3 can be installed using this version: `vue-cascader-select@1.2.0-alpha.0` 😄
